Tools/src/create_rpm.sh

```diff
@@ -47,6 +47,12 @@ chmod ug+rx ${BUILD_ROOT}/SOURCES/opt/aws/amazon-cloudwatch-agent/bin/amazon-clo
 chmod ug+rx ${BUILD_ROOT}/SOURCES/opt/aws/amazon-cloudwatch-agent/bin/start-amazon-cloudwatch-agent
 tar -zcvf ${BUILD_ROOT}/SOURCES/amazon-cloudwatch-agent.tar.gz -C ${BUILD_ROOT}/SOURCES opt etc
 
+echo "*************************************************"
+echo "Saving the tar file for Amazon Linux and RHEL ${ARCH}"
+echo "*************************************************"
+cp ${BUILD_ROOT}/SOURCES/amazon-cloudwatch-agent.tar.gz ${BUILD_SPACE}/bin/linux/${ARCH}/amazon-cloudwatch-agent.tar.gz
+ls -ltr ${BUILD_SPACE}/bin/linux/${ARCH}/*.tar.gz
```

Comment:
I kind of want to move this `ls` call to the bottom after the rpm copy, but I'm not sure right now. I'll think about it more